projects/UnitTest/CMakeLists.txt

```diff
@@ -0,0 +1,150 @@
+# CMakeList.txt : CMake project for XbSymbolDatabaseUnitTest
+#
+cmake_minimum_required(VERSION 3.11)
+
+project(XbSymbolDatabaseUnitTest LANGUAGES CXX)
+
+# Enforce warnings as errors to verify projects are in sync and doesn't have hidden warnings.
+if(${CMAKE_VERSION} VERSION_LESS "3.24.0")
+ if(MSVC)
+  set(COMPILE_WARNING_AS_ERROR "/WX")
+ else()
+  include(CheckCXXCompilerFlag)
+  check_cxx_compiler_flag("-Werror-all" WERRALL)
+  check_cxx_compiler_flag("-Werror" WERR)
+  if(WERRALL)
+   set(COMPILE_WARNING_AS_ERROR "-Werror-all")
+  elseif(WERR)
+   set(COMPILE_WARNING_AS_ERROR "-Werror")
+  else()
+   message(WARNING "Unable enforce compile warnings as errors.")
+  endif()
+ endif()
+ add_compile_options(${COMPILE_WARNING_AS_ERROR})
+else()
+ set(CMAKE_COMPILE_WARNING_AS_ERROR ON)
+endif()
+
+include(FetchContent)
+
+if(NOT TARGET simpleini)
+ FetchContent_Populate(
+  simpleini
+  GIT_REPOSITORY https://github.com/brofield/simpleini.git
+  GIT_TAG        56499b5af5d2195c6acfc58c4630b70e0c9c4c21
+ )
+endif()
+
+if(NOT TARGET xxhash)
+ FetchContent_Populate(
+  xxhash
```

Comment:
Can be installed through vcpkg and probably some other package manager on other platforms.